src/main/java/com/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -39,24 +40,38 @@ public void onCommand(Member sender, CommandContext context) {
         }
         User user = memberForInfo.getUser();
 
+        String status = "";
+
+        if (memberForInfo.getOnlineStatus() == OnlineStatus.ONLINE) {
+            status = context.globalEmote("online");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.OFFLINE) {
+            status = context.globalEmote("offline");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.DO_NOT_DISTURB) {
+            status = context.globalEmote("dnd");
+        } else if (memberForInfo.getOnlineStatus() == OnlineStatus.IDLE) {
+            status = context.globalEmote("idle");
+        } else {
+            status = context.globalEmote("offline");
```

Comment:
Remove this else statement, you've already set it to ""

Reply:
kk xoxo